jobs/settings/.env.dev.sample

```diff
@@ -0,0 +1,3 @@
+; To use dev env ini file.
```

Comment:
Normally the .env file and the .env.dev.sample are in the root of the project and then in the settings you can specify the path of the .env file in the readEnv method.

Reply:
Interesting. I'm going to do it.
Thx

Reply:
Do it! Can you check it again @jbagot , pls?